pandas/tests/resample/test_datetime_index.py

```diff
@@ -1488,3 +1488,13 @@ def test_get_timestamp_range_edges(first, last, offset,
     result = _get_timestamp_range_edges(first, last, offset)
     expected = (exp_first, exp_last)
     assert result == expected
+
+
```

Comment:
this needs to go in pandas/tests/series/test_apply.py

Reply:
Fixed!